AndroidX dependency conflict

Hello,
I am migrating from using Xamarin.Facebook.Login SDK to using this library. I have this error when I try to build my project with this library installed. 

```Error JAVA0000: Error in obj/Debug/lp/133/jl/classes.jar:androidx/core/animation/AnimatorKt$addListener$1.class:
Type androidx.core.animation.AnimatorKt$addListener$1 is defined multiple times: obj/Debug/lp/133/jl/classes.jar:androidx/core/animation/AnimatorKt$addListener$1.class, obj/Debug/lp/172/jl/classes.jar:androidx/core/animation/AnimatorKt$addListener$1.class
Compilation failed
java.lang.RuntimeException: com.android.tools.r8.CompilationFailedException: Compilation failed to complete, origin: obj/Debug/lp/133/jl/classes.jar
androidx/core/animation/AnimatorKt$addListener$1.class
	at com.android.tools.r8.internal.Fj.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:98)
	at com.android.tools.r8.R8.main(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:4)
Caused by: com.android.tools.r8.CompilationFailedException: Compilation failed to complete, origin: obj/Debug/lp/133/jl/classes.jar:androidx/core/animation/AnimatorKt$addListener$1.class
	at Version.fakeStackEntry(Version_3.3.75.java:0)
	at com.android.tools.r8.internal.Fj.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:75)
	at com.android.tools.r8.internal.Fj.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:28)
	at com.android.tools.r8.internal.Fj.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:27)
	at com.android.tools.r8.R8.b(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:1133)
	at com.android.tools.r8.R8.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:2212)
	at com.android.tools.r8.internal.Fj.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:85)
	... 1 more
Caused by: com.android.tools.r8.internal.f: Type androidx.core.animation.AnimatorKt$addListener$1 is defined multiple times: obj/Debug/lp/133/jl/classes.jar:androidx/core/animation/AnimatorKt$addListener$1.class, obj/Debug/lp/172/jl/classes.jar:androidx/core/animation/AnimatorKt$addListener$1.class
	at com.android.tools.r8.internal.JT.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:14)
	at com.android.tools.r8.internal.JT.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:22)
	at com.android.tools.r8.internal.HN.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:33)
	at com.android.tools.r8.internal.HN.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:10)
	at java.base/java.util.concurrent.ConcurrentHashMap.merge(ConcurrentHashMap.java:2048)
	at com.android.tools.r8.internal.HN.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:6)
	at com.android.tools.r8.graph.B2$a.e(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:4)
	at com.android.tools.r8.dex.b.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:110)
	at com.android.tools.r8.dex.b.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:28)
	at com.android.tools.r8.R8.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:902)
	at com.android.tools.r8.R8.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:2211)
	at com.android.tools.r8.internal.Fj.a(R8_3.3.75_b7a6ff6b13548611571508fe72282c9167faa649161ca0013edfc92e19bd7e58:24)
	... 4 more
Directory 'obj/Debug/lp/133' is from 'androidx.core.core-ktx.aar'. (JAVA0000)  java```

--- a/Xamarin.FacebookSlim.Droid/Xamarin.FacebookSlim.Droid.csproj
+++ b/Xamarin.FacebookSlim.Droid/Xamarin.FacebookSlim.Droid.csproj
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <IsPackable>true</IsPackable>
     <PackageId>Xamarin.FacebookSlim.Droid</PackageId>
-    <Version>15.1.0.13</Version>
+    <Version>15.1.0.16</Version>
     <Description>Contains bindings for https://github.com/facebook/facebook-android-sdk</Description>
     <Authors>Christian Kapplmüller, Hipolyto Obeso Huerta</Authors>
     <Company>fun.music IT GmbH</Company>
@@ -66,7 +66,7 @@
     <PackageReference Include="Xamarin.AndroidX.Activity" Version="1.2.0.1" />
     <PackageReference Include="Xamarin.AndroidX.Browser" Version="1.0.0.1" />
     <PackageReference Include="Xamarin.AndroidX.CardView" Version="1.0.0.17" />
-    <PackageReference Include="Xamarin.AndroidX.Core.Core.Ktx" Version="1.5.0" />
+    <PackageReference Include="Xamarin.AndroidX.Core.Core.Ktx" Version="1.9.0.3" />
     <PackageReference Include="Xamarin.AndroidX.Fragment" Version="1.3.0.1" />
     <PackageReference Include="Xamarin.AndroidX.Legacy.Support.Core.Utils" Version="1.0.0.15" />
     <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.15" />

--- a/java/FacebookSlim/facebookslim/build.gradle
+++ b/java/FacebookSlim/facebookslim/build.gradle
@@ -37,7 +37,7 @@ dependencies {
     implementation 'androidx.annotation:annotation:1.1.0'
     implementation 'androidx.legacy:legacy-support-core-utils:1.0.0'
     implementation 'com.android.installreferrer:installreferrer:1.0'
-    implementation 'androidx.core:core-ktx:1.5.0'
+    implementation 'androidx.core:core-ktx:1.9.0'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.cardview:cardview:1.0.0'
     implementation 'androidx.browser:browser:1.0.0'

